packages/internal/test-util/src/lib/spy-driver/spy-driver.module.ts

```diff
@@ -11,13 +13,18 @@ import { SpyDriverConfig } from './spy-driver.config';
  */
 @NgModule()
 export class SpyDriverModule {
-  static forRoot(config?: SpyDriverConfig): ModuleWithProviders<SpyDriverRootModule> {
+  static forRoot(config: Partial<SpyDriverConfig> = {}): ModuleWithProviders<SpyDriverRootModule> {
     return {
       ngModule: SpyDriverRootModule,
       providers: [
         {
           provide: spyDriverConfigToken,
-          useValue: config || {},
+          useFactory: (logDriverConfig: LumberjackLogDriverConfig) => ({
```

Comment:
suggestion: add explicit return type to avoid type errors.
```suggestion
          useFactory: (logDriverConfig: LumberjackLogDriverConfig): LumberjackLogDriverConfig => ({
```